utils/downloads.py

```diff
@@ -56,7 +58,7 @@ def download_sounds(sounds_list, sounds_url=None):
 
     filelist = "%02x %i %s %s\r\n" % (license_crc,
                                     os.stat(tmpf.name).st_size,
```

Comment:
There is still code here regarding a temp file - I thought the reason for doing this change was so that we didn't need a temp file? We should be able to get filesize based on `len(attribution.encode('UTF-8'))`, right?